.github/workflows/ci.yml

```diff
@@ -7,7 +7,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Static checks, unit- and integration tests
+      - name: unit- and integration tests
+        run: |
+         make test-unit
+
+  style:
```

Comment:
As you add a dedicated section here please also extend .mergify.yml in https://github.com/os-autoinst/scripts/blob/master/.mergify.yml#L11 and accordingly https://github.com/os-autoinst/scripts/blob/master/.mergify.yml#L20 with

```patch
       - "status-success~=test"
+      - "status-success~=style"
```

otherwise mergify would merge if tests pass but the style checks which you extended into a separate section could still fail. As an alternative of course you could keep them combined under simply "test". By the way, I would also prefer to not need to explicitly spell out each kind of test in .mergify as this is error prone as soon as we introduce another test job. In os-autoinst and openQA we could go with the alternative to look for the codecov report because only if *all* tests pass then also codecov is called. Maybe eventually we can reach something similar here as well with a fan-in job that combines all results into a final status check that we can check for in mergify

Reply:
Ok. I really like that the different tests are displayed on their own, so it's easy to see where the problem is.

Adding a test that succeeds when all others succeeded should be possible. But maybe for another PR in the future.

Reply:
please also add for https://github.com/os-autoinst/scripts/blob/master/.mergify.yml#L20

Reply:
oh sorry. done